tasks/ssl.yml

```diff
@@ -39,3 +39,14 @@
   - name: Generate secure Diffie Hellman ephemeral parameters
     command: openssl dhparam -dsaparam -out /etc/ssl/certs/dhparam.pem 4096 creates=/etc/ssl/certs/dhparam.pem
   when: site.server.ssl.enabled == True
+
+# is the remote certs to be:
```

Comment:
Adding the `ssl_dir` variable seems to introduced a bit of complexity. I like the idea of having an variable (that can be called `symlink`, for instance) in the site map to indicate whether the remote file should be copied or symlinked.